NoteFind/View Controllers/ProfileViewController.m

```diff
@@ -31,10 +47,70 @@ - (void)setUpUserInfo {
     self.username.text = [NSString stringWithFormat:@"@%@",currentUser[@"username"]];
     self.userSchool.text = currentUser[@"school"];
     self.userMajor.text = currentUser[@"major"];
+        
+}
+
+- (void)getOnlineNotes {
+    PFQuery *noteQuery = [Note query];
+    [noteQuery orderByDescending:@"createdAt"];
+    [noteQuery includeKey:@"author"];
     
+    [noteQuery findObjectsInBackgroundWithBlock:^(NSArray<Note *> *notes, NSError *error) {
+        if (notes) {
+            self.onlineNotes = (NSMutableArray *)notes;
+            [self.collectionView reloadData];
+        } else {
+            [ErrorAlerts showAlertWithTitle:@"couldn't load notes" withMessage:@"failure loading notes. please refresh and try again." withVC:self];
+        }
+    }];
+}
+
+- (void)getOfflineNotes {
+    OfflineDataManager *dataManager = [[OfflineDataManager alloc]init];
+    self.offlineNotes = (NSMutableArray *)[dataManager getOfflineNotes];
+    [self.collectionView reloadData];
+}
+
+
+- (UICollectionViewCell *)collectionView:(nonnull UICollectionView *)collectionView cellForItemAtIndexPath:(nonnull NSIndexPath *)indexPath {
+    if (self.filterSegmentCtrl.selectedSegmentIndex == 0) {
```

Comment:
Can you create integer constants for the segment indexes (0, 1) so its clearer what each one is in this function?

Reply:
done!